### DISCUSSION TITLE
Output logs during rendering

### DISCUSSION THREAD

**Discussion by govindiitd** [pyvista/pyvista#5340] [Q&A] [answered]
Hi,

I am trying to visualize the mesh loaded from openfoam;

mesh = cavity.read()
p1 = pv.Plotter()
p1.add_mesh(mesh,  show_edges=True)
p1.show()

above code doesn't display the mesh, with following error:

C:\Users\ufipc\anaconda3\Lib\site-packages\pyvista\jupyter\notebook.py:33: UserWarning: Failed to use notebook backend: 

No module named 'trame'
Falling back to a static output.
  warnings.warn(
ERROR:root:1: #version 150
ERROR:root:0(135) : error C1503: undefined variable "colorTCoordVCGSOutput"
ERROR:root:Could not set shader program
ERROR:root:1: #version 150
WARNING:root:ERROR: In vtkShaderProgram.cxx, line 428
vtkShaderProgram (0000013DEE445BA0): 

ERROR:root:Could not set shader program
ERROR:root:1: #version 150
WARNING:root:ERROR: In vtkShaderProgram.cxx, line 428
vtkShaderProgram (0000013DEE445BA0): 
ERROR:root:Could not set shader program

However, mesh["internalMesh"] shows the mesh but not mesh["boundaryMesh"].

Other thing; Is it possible to render the graphics in pyvist?

Regards,
Govind

**Comment by tkoyama010** [comment]
It seems that `trame` is not installed in your environment. Please install [`trame`](https://anaconda.org/conda-forge/trame) and try it again.

```bash
conda install -c conda-forge trame
```

If you cannot solve the problem, please run the following code wherever you are experiencing the error, and paste the output in the comment.
This report helps us track down errors and will be critical to addressing your error.

```python
# Get system info
import pyvista as pv
print(pv.Report())
```

**Reply by govindiitd** [reply]
Thanks. I installed but trame version is unknown, see the attached   pv.Report(),
![pv_report](https://github.com/pyvista/pyvista/assets/119282277/422fa876-c62a-4e26-a51a-99d98c57bd9b)

and error: 
![error](https://github.com/pyvista/pyvista/assets/119282277/5e30ee85-c351-4f40-a131-8c75238f48b0)



But I am able to render in 3D mode to some extent.

**Reply by govindiitd** [reply]
pv.Plotter().show()
![mesh_out](https://github.com/pyvista/pyvista/assets/119282277/56921190-bc70-4d81-b3b9-56279e8f312e)

**Reply by tkoyama010** [reply, marked as answer]
Your `pyvista` version is `0.42.3` but the newest version is `0.43.1`. We want to check it in the newest version. Could you please update the version of the packages and report the `pv.Report()` again?

```bash
conda update --all
```

**Reply by banesullivan** [reply]
No idea what is going worng here. However, would you try to uninstall the version of VTK installed from conda and try installing VTK with pip to see if that works? I suspect something is off with the conda-froge Windows build of VTK

**Comment by govindiitd** [comment]
Yeah, It's working now, and able to render the mesh and contours. I have
just updated to the newest version, both pyvista and VTK updated.

When I render, it generates a lot of information and is occupying much
space in notebook which I want to hide. Is there anything that I can do?


Regards,
Govind
